packages/localizations/src/el-GR.ts

```diff
@@ -0,0 +1,692 @@
+  formFieldInputPlaceholder__phoneNumber: '',
+  formFieldInputPlaceholder__username: '',
+  formFieldInputPlaceholder__emailAddress_phoneNumber: '',
+  formFieldInputPlaceholder__emailAddress_username: '',
+  formFieldInputPlaceholder__phoneNumber_username: '',
+  formFieldInputPlaceholder__emailAddress_phoneNumber_username: '',
+  formFieldInputPlaceholder__password: '',
+  formFieldInputPlaceholder__firstName: '',
+  formFieldInputPlaceholder__lastName: '',
+  formFieldInputPlaceholder__backupCode: '',
+  formFieldInputPlaceholder__organizationName: '',
+  formFieldInputPlaceholder__organizationSlug: '',
+  formFieldError__notMatchingPasswords: 'Οι κωδικοί δεν ταιριάζουν.',
+  formFieldError__matchingPasswords: 'Οι κωδικοί ταιριάζουν.',
+  formFieldAction__forgotPassword: 'Ξεχάσατε τον κωδικό;',
+  formFieldHintText__optional: 'Προαιρετικό',
+  formButtonPrimary: 'Συνέχεια',
+  signInEnterPasswordTitle: 'Εισαγωγή κωδικού πρόσβασης',
+  backButton: 'Πίσω',
+  footerActionLink__useAnotherMethod: 'Χρησιμοποιήστε άλλη μέθοδο',
+  badge__primary: 'Κύριο',
+  badge__thisDevice: 'Αυτή η συσκευή',
+  badge__userDevice: 'Συσκευή χρήστη',
+  badge__otherImpersonatorDevice: 'Άλλη συσκευή αντιπροσώπου',
+  badge__default: 'Προεπιλογή',
+  badge__unverified: 'Μη επαληθευμένο',
+  badge__requiresAction: 'Απαιτεί ενέργεια',
+  badge__you: 'Εσείς',
+  footerPageLink__help: 'Βοήθεια',
+  footerPageLink__privacy: 'Προστασία προσωπικών δεδομένων ',
+  footerPageLink__terms: 'Όροι',
+  paginationButton__previous: 'Προηγούμενο',
+  paginationButton__next: 'Επόμενο',
+  paginationRowText__displaying: 'Εμφάνιση',
+  paginationRowText__of: 'από',
+  membershipRole__admin: 'Διαχειριστής',
+  membershipRole__basicMember: 'Μέλος',
+  membershipRole__guestMember: 'Επισκέπτης',
+  signUp: {
+    start: {
+      title: 'Δημιουργήστε τον λογαριασμό σας',
+      subtitle: 'για να συνεχίσετε στο {{applicationName}}',
+      actionText: 'Έχετε ήδη λογαριασμό;',
+      actionLink: 'Σύνδεση',
+    },
+    emailLink: {
+      title: 'Επαληθεύστε το email σας',
+      subtitle: 'για να συνεχίσετε στο {{applicationName}}',
+      formTitle: 'Σύνδεσμος επαλήθευσης',
+      formSubtitle: 'Χρησιμοποιήστε τον σύνδεσμο επαλήθευσης που απεστάλη στη διεύθυνση email σας',
+      resendButton: 'Δεν λάβατε σύνδεσμο; Αποστολή ξανά',
+      verified: {
+        title: 'Επιτυχής εγγραφή',
+      },
+      loading: {
+        title: 'Εγγραφή σε εξέλιξη...',
+      },
+      verifiedSwitchTab: {
+        title: 'Επιτυχής επαλήθευση email',
+        subtitle: 'Επιστροφή στη νέα καρτέλα για να συνεχίσετε',
+        subtitleNewTab: 'Επιστροφή στην προηγούμενη καρτέλα για να συνεχίσετε',
+      },
+    },
+    emailCode: {
+      title: 'Επαληθεύστε το email σας',
+      subtitle: 'για να συνεχίσετε στο {{applicationName}}',
+      formTitle: 'Κωδικός επαλήθευσης',
+      formSubtitle: 'Εισαγάγετε τον κωδικό επαλήθευσης που απεστάλη στη διεύθυνση email σας',
```

Comment:
@EmmanouelaPothitou I think "Εισάγετε τον κωδικό επαλήθευσης που απεστάλη στο email σας" is better. WDYT?

or alternatively
"Εισάγετε τον κωδικό επαλήθευσης που απεστάλη σε αυτή την διεύθυνση"